src/a-d/Alma/Alma.BufferList.cs

```diff
@@ -45,6 +45,45 @@ public AlmaList(
         _norm = norm;
     }
 
+    /// <summary>
+    /// Initializes a new instance of the <see cref="AlmaList"/> class with initial quotes.
+    /// </summary>
+    /// <param name="lookbackPeriods">The number of periods to look back for the calculation.</param>
+    /// <param name="offset">The offset for the ALMA calculation. Default is 0.85.</param>
+    /// <param name="sigma">The sigma for the ALMA calculation. Default is 6.</param>
+    /// <param name="quotes">Initial quotes to populate the list.</param>
+    public AlmaList(
+        int lookbackPeriods,
+        double offset,
+        double sigma,
+        IReadOnlyList<IQuote> quotes)
+    {
+        Alma.Validate(lookbackPeriods, offset, sigma);
```

Comment:
It’d be less duplicative to call the other constructor, right?  Potential refactor in many of these.